lib/converter.js

```diff
@@ -88,13 +86,26 @@ async function convert (processor, inputFile, options, timings) {
     if (format) {
       pdfOptions.format = format // Paper format. If set, takes priority over width or height options. Defaults to 'Letter'.
     }
-    const pdf = await page.pdf(pdfOptions)
-    if (outputToStdout) {
-      console.log(pdf.toString())
+    let pdf = debug || await page.pdf(pdfOptions)
+    if (!debug) {
+      if (doc.getAttribute('toc') !== undefined) {
+        pdf = await addOutline(pdf, doc)
```

Comment:
I think we should add a comment with the Chromium issue.
Something like:

> Outline is not yet implemented in Chromium:
> https://bugs.chromium.org/p/chromium/issues/detail?id=840455

Reply:
good idean